sdk/deploymentmanager/mgmt-v2019-11-01-preview/pom.xml

```diff
@@ -0,0 +1,135 @@
+<!--
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.microsoft.azure.deploymentmanager.v2019_11_01_preview</groupId>
+    <parent>
+        <groupId>com.microsoft.azure</groupId>
+        <artifactId>azure-arm-parent</artifactId>
+        <version>1.1.0</version>
```

Comment:
/CC: @weidongxu-microsoft 
com.microsoft.azure:azure-arm-parent (pom.management.xml)'s version is currently 1.2.0, shouldn't this be referencing that version and not the older, 1.1.0, version?

Reply:
Yes, Tanyi please fix it.

This is due to latest autorest.java extension not yet published, else it would be 1.2.0 automatically. So at present we had to do it manually.

I think Tanyi is now trying to double check with service team on that `RolloutRequestInner` in **response** body...

Reply:
Done